backend/app.py

```
@@ -36,7 +36,7 @@ def price():
     product_id = request.args.get("id", "0")
     conn = get_db()
     cur = conn.cursor()
-    cur.execute("SELECT price FROM products WHERE id=" + product_id)
+    cur.execute("SELECT price FROM products 12 WHERE id=" + product_id)
```

Ruff (0.14.2)

Possible SQL injection vector through string-based query construction

(S608)